### ISSUE TITLE
How to get logit value from detection mode

### ISSUE THREAD

**Issue by Hazzng** [ultralytics/yolov5#9712]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hi i want to take the logit value before softmax of yolov5 but i wonder how can we do it in detection mode. Here is my code 

```
model = torch.hub.load('yolov5', 'custom', path="/home/callum/Backdoored-Detector/Weights/BLUE_LOW_SCALING_P0.1.pt", source='local')
img = cv2.imread("1___xaqDe9h8QfOyxTt0224Q.jpg")
thisResult = model(img[:,:,::-1])
# How to take logit from this result object -> i have searched and can't find the logit inside this object. 
#I wonder are there any ways to get the logit if i only put one image to model
print(thisResult.xyxy[0]
```

**Comment by glenn-jocher** [comment]
@QuangNguyen2609 outputs are sigmoided here in the Detect() module, you might be able to insert code to handle these values however you want:
https://github.com/ultralytics/yolov5/blob/7f097ddb6c9921d64fa504a8db79cf24fa0a913c/models/yolo.py#L72-L73

**Comment by QuangNguyen2609** [comment]
Thanks ! Im still wondering putting one image like this will use Detection mode and Autoshape as well but why the model in Autoshape only returns one value (y) 


```
with amp.autocast(enabled=p.device.type != 'cpu'):
    # Inference
    y = self.model(x, augment, profile)[0]  # forward
    print("AUTOSHAPE: ", y.shape)
    t.append(time_sync())
```

```
print("DETECT HEAD: ", torch.cat(z, 1).shape)
return x if self.training else (torch.cat(z, 1), x)
# Detect model returns 2 values 
```
